rotkehlchen/exchanges/exchange.py

```diff
@@ -230,14 +230,31 @@ def query_online_income_loss_expense(
         Has to be implemented by exchanges if they have anything exchange specific
 
         For example coinbase
+
+        Deprecated, will be replaced by query_online_history_events
```

Comment:
Hm ... ehm ... hmm ... who uses this? I did a really quick check and it seems unused? If so you can remove the entire thing in a different commit too.

Reply:
Coinbase was the only exchange using this function at all, and it was easily converted to use `query_online_history_events` instead, so I've added a commit removing `query_income_loss_expense` and `query_online_income_loss_expense` completely.